scripts/build_gap.sh

```diff
@@ -2,7 +2,8 @@
 set -ex
 
 # clone GAP into a subdirectory
-git clone --depth=2 -b ${GAPBRANCH:-master} https://github.com/gap-system/gap.git $GAPROOT
+# HACK HACK HACK FOR TRAVIS TESTING
+git clone --depth=2 -b fork-fix-2 https://github.com/ChrisJefferson/gap.git $GAPROOT
```

Comment:
What's this for? Perhaps to make things work until your GAP PR was merged? But that has happened now, so, can this be removed?

Reply:
I would have added a 'do not merge', but I don't seem to be able to add tags (and it didn't seem worth figuring out why that much)

Reply:
@ChrisJefferson I guess you mean labels, not tags. Created 'do not merge' for you, you can now apply it if it's still relevant.